03_Modules/Configuration/src/module/module.ts

```diff
@@ -574,11 +601,19 @@ export class ConfigurationModule extends AbstractModule {
   }
 
   @AsHandler(CallAction.SetDisplayMessage)
-  protected _handleSetDisplayMessage(
-    message: IMessage<SetDisplayMessageResponse>,
-    props?: HandlerProperties
-  ): void {
+  protected async _handleSetDisplayMessage(
+      message: IMessage<SetDisplayMessageResponse>,
+      props?: HandlerProperties
+  ): Promise<void> {
     this._logger.debug("SetDisplayMessage response received:", message, props);
+
+    const status = message.payload.status as DisplayMessageStatusEnumType;
+    // when charger station accepts the set message info request
+    // we trigger a get all display messages request to update stored message info in db
```

Comment:
Since in the [SetDisplayMessageResponse](https://github.com/citrineos/citrineos-core/blob/rc-1.1.0/00_Base/src/ocpp/model/types/SetDisplayMessageResponse.ts) and clear display message response there is no any MessageInfoType identifiers but only status, we cannot know which messageInfo is accepted/rejected. So I trigger the get all message info after every set/clear request and save messageInfos only in the get message info handler.

Reply:
I see--there's no easy way to correlate the two messages. We've run into similar issues in the past with VariableAttributes and VariableMonitoring. I'm going to make a discussion topic after 1.1.0 release to delve into what solutions might make sense. This is fine for now.